lib/engines/content_block_manager/features/create_contact_object.feature

```diff
@@ -3,17 +3,51 @@ Feature: Create a contact object
   Background:
     Given I am a GDS admin
     And the organisation "Ministry of Example" exists
-    And a schema "contact" exists with the following fields:
-      | field         | type   | format | required |
-      | description   | string | string | true     |
-    And the schema "contact" has a subschema with the name "email_addresses" and the following fields:
-      | field            | type   | format | required | enum           | pattern          |
-      | title            | string | string | true     |                |                  |
-      | email_address    | string | string | true     |                |                  |
-    And the schema "contact" has a subschema with the name "telephones" and the following fields:
-      | field     | type   | format | required | enum           | pattern          |
-      | title     | string | string | true     |                |                  |
-      | telephone | string | string | true     |                |                  |
+    And a schema "contact" exists:
+    """
+    {
```

Comment:
I like this change, it might be a bit more fiddly to setup but at least it makes it more transparent what we are actually doing. I guess if needed we could go one step further and have the json strings saved somewhere so we could re-use them?

Reply:
I did wonder that, but I wasn't sure if it'd make it less visible? We could give it a go though if it gets unweildy